zetaforge/forge_runner.py

```diff
@@ -309,12 +310,27 @@ def teardown(driver):
 
     print("Completed teardown!")
 
-def uninstall():
-    install = subprocess.run(["kubectl", "delete", "-f", INSTALL_YAML], capture_output=True, text=True)
-    print ("Removing install: ", {install.stdout})
-    build = subprocess.run(["kubectl", "delete", "-f", BUILD_YAML], capture_output=True, text=True)
```

Comment:
I get the desire to remove this but the reason for this is to be able to remove version 0.1.2 registry. We cannot guarantee that any particular user doesn't have that version, so we need to be able to keep `build-0.1.2.yaml` in order to delete it. We will have this problem whenever we change the services and likely need to maintain these files like migrations.

Reply:
My main issue here is not the yaml file but rather not being tied to kubectl. Moving forward, we can track the yaml files as migrations, but because of how the pip package is currently we cannot be retrocompatible with those files.